### DISCUSSION TITLE
Input widgets take over shortcuts

### DISCUSSION THREAD

**Discussion by hwjsnc** [fltk-rs/fltk-rs#1394] [General]
I've registered the shortcut Ctrl+z to a menu item to undo the last change to global app state. When an input widget is focused, the shortcut is instead handled by that input widget which undoes the last text change to the input widget. Is there any way to stop that from happening and only have the global shortcut?

**Comment by MoAlyousef** [comment]
Not with the Rust bindings. Even with C++, you would have to avoid calling the the input's default handle, then handle all key input manually.

**Comment by hwjsnc** [comment]
That's unfortunate. As a workaround I decided to change the global shortcut to something else. Thanks for the help!
